HLTrigger/JetMET/plugins/BuildFile.xml

```diff
@@ -14,6 +14,7 @@
   <use   name="FWCore/Utilities"/>
   <use   name="RecoMET/METAlgorithms"/>
   <use   name="HLTrigger/HLTcore"/>
+  <use   name="HLTrigger/JetMET"/>
```

Comment:
It seems to me that you need to move all the new files into the ```plugins``` directory. That will make your module available to the framework.